docs/kuraKapuaDocs.md

```diff
@@ -84,7 +84,7 @@ kapua-password
 
 ### Adding Account and User
 
-Now you can login into Kapua, but we still need to create an Account and a User. This section has for steps. First one is creating a child account. 
+Now you can login to Kapua, but we still need to create an Account and a User. This section has four steps. The first one is creating a child account. 
```

Comment:
"The first one" is not gramatically correct. Please change it back to "First one...."